plugins/kn-image/README.md

```diff
@@ -0,0 +1,220 @@
+# Kn service build and deploy plugin
+
+## Prequirement
+Before you begin this task, you should prepare a kubernetes cluster which has been installed tekton-pipeline and access it.
+### Confirm tekton-pipeline has been installed
+```
+> kubectl get pod -n tekton-pipelines
+NAME                                           READY   STATUS    RESTARTS   AGE
+tekton-pipelines-controller-6d8f8c4f66-6lcwf   1/1     Running   0          5d4h
+tekton-pipelines-webhook-5697f86c7c-d5c9r      1/1     Running   0          3d
+tekton-triggers-controller-5f58c5775f-zshvf    1/1     Running   0          5d4h
+tekton-triggers-webhook-74497cb8dd-h9qxd       1/1     Running   0          3d
+```
+Ensure all pods are `running`.
+
+## Create build task first
+Please create the build task first from https://github.com/tektoncd/catalog
+### Such as buildpacks build task:
+- Download yaml from https://github.com/tektoncd/catalog/blob/master/buildpacks/buildpacks-v3.yaml as buildpacks-v3.yaml
+- `kubectl apply -f buildpacks-v3.yaml`
+You can use `kubectl get task` command to see the task. It's output should like below:
+```
+NAME            AGE
+buildpacks-v3   4d21h
+```
+### Such as kaniko build task:
+- Download yaml from https://github.com/tektoncd/catalog/blob/master/kaniko/kaniko.yaml
+- `kubectl apply -f kaniko.yaml`
+
+## Prepare secrets
```

Comment:
Don’t you need also a Docker image repository Account too?

Reply:
Yes, the Docker account (username and password) is the prerequisite